bin/node-lambda

```diff
@@ -120,6 +120,9 @@ program
   .command('run')
   .alias('execute')
   .description('Run your Amazon Lambda application locally')
+  .option('-a, --accessKey [' + AWS_ACCESS_KEY_ID + ']', 'AWS Access Key', AWS_ACCESS_KEY_ID)
+  .option('-s, --secretKey [' + AWS_SECRET_ACCESS_KEY + ']', 'AWS Secret Key', AWS_SECRET_ACCESS_KEY)
+  .option('-P, --profile [' + AWS_PROFILE + ']', 'AWS Profile', AWS_PROFILE)
```

Comment:
This makes no sense here, `run` is local only... perhaps you can explain the usecase a bit better?

Reply:
Hey @DeviaVir...good question...for sure the event.js is run against the handler locally.

But what happens _inside_ the handler often interacts with AWS services (it may publish to kinesis or upload to S3, etc)- and being able to pass credentials and profile in a familiar way (e.g. `--profile`) sounds like the right the thing to do. 

That's a pretty common use for the lambdas we're deploying.
Let me know if that answers your question.

Reply:
What stops you from using
```
AWS_PROFILE=
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
``` 
environment variables?

The part I'm unsure about here, is that for a handler, you'd just expect it to be agnostic to what it runs. If we start passing these vars, the handler is not agnostic anymore.

Reply:
Hey @DeviaVir - good question, sorry for the late reply.

## Agnosticism of handlers

Hmmmm - like I said in [my prior comment](https://github.com/motdotla/node-lambda/pull/379#discussion_r143000897) - our handlers are rarely an island unto themselves.
They mostly do _something_ aws-y as a result...like publishing to kinesis or uploading to S3.

## Using AWS_PROFILE, AWS_ACCESS_KEY_ID, and AWS_SECRET_ACCESS_KEY

When you talk about adding those three env vars - you're talking about adding them to `.env` right?

`node-lambda` favors having one AWS account across deployments. There's a lot of parallel flexibility between `deploy` and `run` now (e.g: `--runtime`, `--timeout`, `--configFile`) and I feel that adding `--profile` adds to that flexibility.

Without the parameter, developers have to juggle the `.env` files depending on which account's services they want the handler to interact with. At least in our team, that juggling has been too easy to mess up. Being able to control it with `--profile` would be more wieldy than juggling `.env`s.